src/main/java/org/apache/ibatis/jdbc/AbstractSQL.java

```diff
@@ -360,27 +373,28 @@ public boolean isEmpty() {
     }
 
     StatementType statementType;
-    List<String> sets = new ArrayList<>();
```

Comment:
Changes of 3502f7ced8ec30f6196c4253df4d22acba601c40 (merging from latest master branch by @harawata) was reverted. Probably your merging operation is incorrect. In this case, the code that uses the diamond operator is latest and correct implementation.

Reply:
@sivdead This comment does not fixed. Why?

Reply:
@kazuki43zoo please recheck it ,maybe i used an wrong branch

Reply:
@sivdead I've fixed it and merged at now! Thanks for your contributing!